LayoutTests/fast/css/resize-mutation-observer.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<script>
+function resize(id) {
+  const target = document.getElementById("div");
+  const result = document.getElementById("result");
+
+  function handleResize (records) {
+    for (const r of records) {
+      result.textContent += `\n\n${r.type}, ${r.attributeName}, ${r.oldValue}`;
+    }
+    window.testRunner.dumpAsText(result.textContent);
```

Comment:
dumpAsText() does not take arguments, you just call it in the beginning and all the text content gets dumped